tests/evergreen/test_project.py

```diff
@@ -0,0 +1,45 @@
+# -*- encoding: utf-8 -*-
+"""Unit tests for src/evergreen/project.py."""
+from __future__ import absolute_import
+
+from unittest.mock import MagicMock
+
+from evergreen.project import Project
+
+
+class TestProject(object):
+    def test_most_recent_version(self, sample_project):
+        mock_api = MagicMock()
+        project = Project(sample_project, mock_api)
+        mock_api.versions_by_project.return_value = (
+            MagicMock(version_id=f"version_{i}") for i in range(3)
+        )
+
+        most_recent_version = project.most_recent_version()
+
+        assert most_recent_version.version_id == "version_0"
+
+    def test_module_by_name(self, sample_project):
+        mock_api = MagicMock()
```

Comment:
[nit] There are a lot of mocks being created here for a fairly small test. That makes me think something is up.

Reply:
Agreed, once I moved this method to the `Version` class, the test got a lot cleaner.